projects/adventure/adv.py

```diff
@@ -17,14 +18,77 @@
 world.loadGraph(roomGraph)
 
 # UNCOMMENT TO VIEW MAP
-world.printRooms()
+# world.printRooms()
 
 player = Player("Name", world.startingRoom)
 
 # Fill this out
 traversalPath = []
+path_by_ids = []
+graph = Graph()
+visited_rooms = set()
+identified_rooms = set()
+dir_stack = Stack()
+
+#Need a way to back track when reach deadends
+def reverse_dir(direction):
```

Comment:
nice use making a function for this.  Similarly, we could create a hash table / dictionary to do this same job.